tests/concurrency.py

```diff
@@ -15,17 +15,19 @@
 
 
 @functools.singledispatch
-async def sleep(concurrency_backend: ConcurrencyBackend, seconds: int) -> None:
+async def sleep(concurrency_backend: ConcurrencyBackend, seconds: float) -> None:
```

Comment:
Linting was complaining because of https://github.com/florimondmanca/asgi-lifespan/blob/650022b7b794d3d1573c6ae833a647b89e378bd9/tests/test_manager.py#L104 and other calls